src/types.jl

```diff
@@ -1,4 +1,4 @@
-struct ArVar{Ti <: Integer}
+@kwdef struct ArVar{Ti <: Integer}
```

Comment:
This breaks v1.6 CI as at that time the macro was not exported. 
```suggestion
@Base.kwdef struct ArVar{Ti <: Integer}
```

Reply:
I was writing the comment but you have been faster to correct it.